omod/src/main/resources/defaultNarratives/AllergyIntolerance.html

```diff
@@ -0,0 +1,153 @@
+                <td th:text="${#dates.format(resource.assertedDate,'dd/MM/yyyy hh:mm:ss a')}"></td>
+            </tr>
+        </th:block>
+        <th:block th:unless="${fhirVersion} eq 'DSTU3'">
+            <tr th:if="${resource.recordedDate}">
+                <td>Recorded Date:</td>
+                <td th:text="${#dates.format(resource.recordedDate,'dd/MM/yyyy hh:mm:ss a')}"></td>
+            </tr>
+        </th:block>
+        <tr th:if="${not resource.recorder.empty}">
+            <td>Recorder:</td>
+            <td th:narrative="${resource.recorder}"></td>
+        </tr>
+        <tr th:if="${not resource.asserter.empty}">
+            <td>Asserter:</td>
+            <td th:narrative="${resource.asserter}"></td>
+        </tr>
+        <tr th:if="${resource.lastOccurrence}">
+            <td>Last Occurrence:</td>
+            <td th:text="${#dates.format(resource.lastOccurrence,'dd/MM/yyyy')}"></td>
+        </tr>
+        <tr th:if="${not resource.note.empty}">
+            <td>Note:</td>
+            <td>
+                <div th:each="note: ${resource.note}" th:text="${note.text}"></div>
+            </td>
+        </tr>
+        </tbody>
+    </table>
+    <th:block th:each="reaction: ${resource.reaction}">
+        <br/>
+        <table class="hapiPropertyTable">
+            <tbody>
+            <tr>
+                <td>Reaction:</td>
+                <td></td>
+            </tr>
+            <tr th:if="${not reaction.substance.empty}">
+                <td>Substance:</td>
+                <td th:narrative="${reaction.substance}"></td>
+            </tr>
+            <tr th:if="${not reaction.manifestation.empty}">
+                <td>Manifestation:</td>
+                <td>
+                    <th:block th:each="manifestation: ${reaction.manifestation}">
+                        <div th:if="${not manifestation.empty}" th:narrative="${manifestation}"></div>
+                    </th:block>
+                </td>
+            </tr>
+            <tr th:if="${reaction.description}">
+                <td>Description:</td>
+                <td th:text="${reaction.description}"></td>
+            </tr>
+            <tr th:if="${reaction.onset}">
+                <td>Onset:</td>
+                <td th:text="${#dates.format(reaction.onset,'dd/MM/yyyy')}"></td>
+            </tr>
+            <tr th:if="${reaction.severity}">
+                <td>Severity:</td>
+                <td th:text="${reaction.severity}"></td>
+            </tr>
+            <tr th:if="${not reaction.exposureRoute.empty}">
+                <td>Exposure Route:</td>
+                <td th:narrative="${reaction.exposureRoute}"></td>
+            </tr>
+            <tr th:if="${not reaction.noteFirstRep.empty}">
+                <td>Note:</td>
+                <td th:text="${reaction.noteFirstRep.text}"></td>
+            </tr>
+            </tbody>
+        </table>
+    </th:block>
+</div>
+
+<!--/*-->
+</body>
+
+</html>
+<!--*/-->
```

Comment:
Always have a new line at the end of the file. This is primarily to prevent some Git weirdness issues.